src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -510,15 +518,18 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
       stay using finalData
     case Event(PersistenceSucceeded(callKey, ExecutionStatus.Preempted, callOutputs), data) =>
       executionStore += callKey -> ExecutionStatus.Preempted
-      val updatedData = startRunnableCalls(data)
-      val finalData = updatedData.removePersisting(callKey, ExecutionStatus.Preempted)
-      self ! CheckForWorkflowComplete
-      stay using finalData
+      val updatedData = retryCall(callKey, data).removePersisting(callKey, ExecutionStatus.Preempted)
```

Comment:
Oh yeah this one is a `removePersisting(callKey, ExecutionStatus.NotStarted)`
It's the mechanism used to keep track of what is currently being persisted and avoid out of order status persistence. I think you're right that in those cases it doesn't really matter as nothing really cares if something is being persisted to `Preempted` or `NotStarted`, I just added it for consistency.